src/trackedit/internal/au3/au3selectioncontroller.cpp

```diff
@@ -81,62 +84,18 @@ void Au3SelectionController::resetDataSelection()
 {
     MYLOG() << "resetDataSelection";
 
-    auto& tracks = ::TrackList::Get(projectRef());
-    for (trackedit::TrackId trackId : m_dataSelectedTrackIds.val) {
-        ::Track* au3Track = tracks.FindById(::TrackId(trackId));
-        if (au3Track) {
-            au3Track->SetSelected(false);
-        }
-    }
-
-    m_dataSelectedTrackIds.set(TrackIdList(), true);
     m_selectedStartTime.set(-1.0, true);
     m_selectedEndTime.set(-1.0, true);
 }
 
-bool Au3SelectionController::isDataSelected() const
+bool Au3SelectionController::timeSelectionIsNotEmpty() const
 {
-    return muse::RealIsEqualOrMore(m_selectedStartTime.val, 0.0) && m_selectedEndTime.val > 0.0;
+    return muse::RealIsEqualOrMore(m_selectedStartTime.val, 0.0) && m_selectedEndTime.val > 0.0 && muse::RealIsEqualOrMore(m_selectedEndTime.val, m_selectedStartTime.val);
```

Comment:
I think that last check must `m_selectedEndTime.wav > m_selectedStartTime.val`, shouldn't it? i.e. if they are equal, this method should return false - I think.

Reply:
I'm more confident now: `doGlobalCut()`, `doGlobalDelete()` and other such should check this, and be no-opts if the selection is just a point selection.